ProjectSettings/ProjectSettings.asset

```diff
@@ -3,7 +3,7 @@
 --- !u!129 &1
 PlayerSettings:
```

Comment:
I'm not loving all the changes being made to player settings here. I'm thinking we should revert all of this + quality settings before merge.

Reply:
Restored.